src/metric-engine/src/engine.rs

```diff
@@ -587,23 +609,36 @@ impl MetricEngineInner {
             }
         }
 
-        let data_region_id = utils::to_data_region_id(region_id);
+        let data_region_id = utils::to_data_region_id(physical_region_id);
         self.add_columns_to_physical_data_region(
             data_region_id,
             metadata_region_id,
-            logical_table_id,
+            region_id,
             columns_to_add,
         )
         .await?;
 
         Ok(())
     }
+
+    async fn alter_physical_region(
+        &self,
+        region_id: RegionId,
+        request: RegionAlterRequest,
+    ) -> Result<()> {
+        info!("Metric region received alter request {request:?} on physical region {region_id:?}");
```

Comment:
I think we can allow this operation in future for administration. For example, deleting some unused columns.

Reply:
We can allow more operations to the physical region/table in the future. It's currently limited to read-only.